### PR TITLE
add license and fix problems from moving repo

### PR DESCRIPTION
**Pull request by fuchsnj**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) [year] [fullname]
```
Comment (jagoda) [inline LICENSE]: This should have the values substituted in. Something like `Copyright (c) 2015 Bandwidth.com`.

**Comment by wtcross** [comment]
How about providing a link to the license from the readme?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 #Bandwidth WebRTC Client
-[![Build Status](https://magnum.travis-ci.com/inetCatapult/webrtc-client.svg?token=BQBnkCKjpv5Ls9SpJgzy&branch=master)](https://magnum.travis-ci.com/inetCatapult/webrtc-client)
-[![Coverage Status](https://coveralls.io/repos/inetCatapult/webrtc-client/badge.png?branch=master)](https://coveralls.io/r/inetCatapult/webrtc-client?branch=master)
+[![Build Status](https://magnum.travis-ci.com/bandwidthcom/webrtc-client.svg?token=BQBnkCKjpv5Ls9SpJgzy&branch=master)](https://magnum.travis-ci.com/bandwidthcom/webrtc-client)
```
Comment (jagoda) [inline README.md]: Should this still be `magnum.travis-ci.com`? I think that we would want/need to move it to `travis-ci.org` (basically paid Travis vs. free Travis). I suspect that the webhook needs to be reconfigured as well.

Comment (wtcross) [inline README.md]: It should be `travis-ci.org`

**Comment by wtcross** [comment]
Looks good

**Pull request merged**